.circleci/dynamic_config.yml

```diff
@@ -283,16 +296,23 @@ jobs:
     steps:
       - custom_attach_workspace
       - run: yarn build
+      - persist_to_workspace:
+          root: *workspace_location
+          paths:
+            - dist/_*.tgz
 
-  build-bazel-e2e:
-    executor: action-executor
+  bazel-build:
+    executor: bazel-executor
```

Comment:
This would cause usage `2xlarge` resource_class class which is not great.I believe here large should be enough.

Reply:
This was switched to just `xlarge` and I think we confirmed upgrading the resource_class was ok so I'm going to resolve this for now.